readme.md

```diff
@@ -0,0 +1,22 @@
+# Telegram bot sandbox
```

Comment:
good `README.md` should also mention the stuff like package installation, the purpose of the project, etc. (or where to find all of these). However, this exact repo is a small training one, so it doesn't matter.

Reply:
oh, nvm, I see now, it's covered somehow.